sources/packages/api/src-sql/EducationPrograms/Alter-education-programs.sql

```diff
@@ -0,0 +1,159 @@
+--Drop the NOT NULL constraint on program_description
+ALTER TABLE
+    sims.education_programs
+ALTER COLUMN
+    program_description DROP NOT NULL;
+
+--Changing the data type of has_minimun_age as it is one of entrance requirement now.
+ALTER TABLE
+    sims.education_programs
+ALTER COLUMN
+    has_minimun_age TYPE BOOLEAN USING CASE
+        has_minimun_age
+        WHEN 'yes' THEN TRUE
+        ELSE FALSE
+    END;
+
+-- Add institution_program_code to education_programs table
+ALTER TABLE
+    sims.education_programs
+ADD
+    COLUMN IF NOT EXISTS institution_program_code VARCHAR(50);
+
+COMMENT ON COLUMN sims.education_programs.institution_program_code IS 'Institution program code of the given program';
+
+-- Add min_hours_week to education_programs table
+ALTER TABLE
+    sims.education_programs
+ADD
+    COLUMN IF NOT EXISTS min_hours_week VARCHAR(50);
+
+COMMENT ON COLUMN sims.education_programs.min_hours_week IS 'This column represents result for the question, Does this program include a minimum of 20 instructional hours per week? ';
```

Comment:
Please remove the white space at the end of the comment.

Reply:
Done.